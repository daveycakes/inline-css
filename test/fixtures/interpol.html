<!DOCTYPE html>
<html>
	<head>
	<title><%= @rails %></title>
	<style>
	h1{
		line-height: 2em;
	}
	p{
		color: #123;
		font-weight: bold;
	}
	</style>
	</head>
	<body>
		<h1><<LCG Program\TG LCG Coupon Code Default='246996'>></h1>
		<p style="color: {{protected}};">Woo ha</p>
		<p>Just a paragraph</p>
	</body>
</html>