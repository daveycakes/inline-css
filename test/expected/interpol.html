<!DOCTYPE html>
<html>
	<head>
	<title><%= @rails %></title>
	
	</head>
	<body>
		<h1 style="line-height: 2em;"><<LCG Program\TG LCG Coupon Code Default='246996'>></h1>
		<p style="color: {{protected}}; font-weight: bold;">Woo ha</p>
		<p style="color: #123; font-weight: bold;">Just a paragraph</p>
	</body>
</html>